ansible/files/home/packages/windows/groonga/.htaccess

```diff
@@ -0,0 +1,2 @@
+RewriteRule "groonga-latest-(.+?)-vs(.+?)\.zip" "https://github.com/groonga/groonga/releases/download/v12.0.8/groonga-12.0.8-$1-$2.zip"
```

Comment:
I think it's not good to specify a literal value for `groonga-latest` but I didn't get good ideas for it...

Anyone have any good ideas?

Reply:
Hmm, I should consider the case that vs2015 case...